src/System.Drawing.Common/src/System/Drawing/NativeStructs.Unix.cs

```diff
@@ -54,7 +54,7 @@ internal struct LOGFONT
         internal string lfFaceName;
     }
 
-    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+    [StructLayout(LayoutKind.Sequential)]
     internal struct GdipImageCodecInfo  /*Size 76 bytes*/
```

Comment:
Nit: You can also delete "/*Size 76 bytes*/" command while you are on it. It is not very useful and it is wrong for 64-bit platforms.